.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Shouldn't the env vars be similar to core?

Reply:
The `SKIP_PREFLIGHT_CHECK=true` is necessary to let Create React App (what I've used to kickstart this boilerplate) know that it's okay that we're using a different eslint version than the one it initially installs.

As for the remaining environment variables, I've now added them in.